.github/workflows/workflow.yml

```diff
@@ -0,0 +1,285 @@
+          cd bitcoinfuzz
+          export CXXFLAGS="-DBITCOIN_CORE -DRUST_MINISCRIPT -DEMBIT"
+          make
+          if [ -d "../descriptor_parse" ]; then
+            echo "Using corpora for descriptor_parse"
+            ASAN_OPTIONS=detect_leaks=0 FUZZ=descriptor_parse ./bitcoinfuzz -runs=1 ../descriptor_parse
+          else
+            echo "No corpora found for descriptor_parse, using runs=100"
+            ASAN_OPTIONS=detect_leaks=0 FUZZ=descriptor_parse ./bitcoinfuzz -runs=100
+          fi
+
+      - name: Test - miniscript_parse
+        run: |
+          cd bitcoinfuzz
+          export CXXFLAGS="-DBITCOIN_CORE -DRUST_MINISCRIPT -DEMBIT"
+          make
+          if [ -d "../miniscript_parse" ]; then
+            echo "Using corpora for miniscript_parse"
+            ASAN_OPTIONS=detect_leaks=0 FUZZ=miniscript_parse ./bitcoinfuzz -runs=1 ../miniscript_parse
+          else
+            echo "No corpora found for miniscript_parse, using runs=0"
+            ASAN_OPTIONS=detect_leaks=0 FUZZ=miniscript_parse ./bitcoinfuzz -runs=100
+          fi
+
+      - name: Test - script_asm
+        run: |
+          cd bitcoinfuzz
+          export CXXFLAGS="-DBITCOIN_CORE -DBTCD"
+          make
+          if [ -d "../script_asm" ]; then
+            echo "Using corpora for script_asm"
+            FUZZ=script_asm ./bitcoinfuzz -runs=1 ../script_asm
+          else
+            echo "No corpora found for script_asm, using runs=0"
+            FUZZ=script_asm ./bitcoinfuzz -runs=100
+          fi
+
+      - name: Test - address_parse
+        run: |
+          cd bitcoinfuzz
+          export CXXFLAGS="-DBITCOIN_CORE -DRUST_BITCOIN"
+          make
+          if [ -d "../address_parse" ]; then
+            echo "Using corpora for address_parse"
+            FUZZ=address_parse ./bitcoinfuzz -runs=1 ../address_parse
+          else
+            echo "No corpora found for address_parse, using runs=100"
+            FUZZ=address_parse ./bitcoinfuzz -runs=100
+          fi
+
+      - name: Test - psbt_parse
+        run: |
+          cd bitcoinfuzz
+          export CXXFLAGS="-DEMBIT -DRUST_BITCOIN -DBTCD"
+          make
+          if [ -d "../psbt_parse" ]; then
+            echo "Using corpora for psbt_parse"
+            ASAN_OPTIONS=detect_leaks=0 FUZZ=psbt_parse ./bitcoinfuzz -runs=1 ../psbt_parse
+          else
+            echo "No corpora found for psbt_parse, using runs=100"
+            ASAN_OPTIONS=detect_leaks=0 FUZZ=psbt_parse ./bitcoinfuzz -runs=100
+          fi
+
+      - name: Test - deserialize_invoice
+        run: |
+          cd bitcoinfuzz
+          export CXXFLAGS="-DLDK -DLND -DCLIGHTNING -DCUSTOM_MUTATOR_BOLT11"
+          (cd ./modules/bitcoin && make clean)
+          make
+          if [ -d "../deserialize_invoice" ]; then
+            echo "Using corpora for deserialize_invoice"
+            ASAN_OPTIONS=detect_leaks=0 FUZZ=deserialize_invoice ./bitcoinfuzz -runs=1 ../deserialize_invoice
+          else
```

Comment:
I'm not sure about the approach. I think the goal is to check the coverage so if there is no corpora we could simply skip it instead of using `-runs=100`.